src/MUST_analysis_tools/MUST_CP_calibration_analysis.py

```diff
@@ -0,0 +1,867 @@
+        # gps_x_e1, gps_y_e1 = map(gps_data['longitude'] + x_above, gps_data['latitude'] + y_above)
+        # gps_x_e2, gps_y_e2 = map(gps_data['longitude'] + x_below, gps_data['latitude'] + y_below)
+
+        # Plot the data points
+        map.plot(must_x, must_y, markersize=5, label=f'MUST calibrated track')
+        map.plot(gps_x, gps_y, markersize=5, label=f'GPS track')
+        map.plot(must_uw_x, must_uw_y, markersize=5, label=f'MUST UW track')
+        map.plot(must_x_smoothed, must_y_smoothed, markersize=5, label=f'MUST calibrated smoothed')
+        # map.plot(must_local_x, must_local_y, markersize=5, label=f'MUST track')
+        # map.plot(gps_x_e1, gps_y_e1, markersize=5, label=f'GPS error bars')
+        # map.plot(gps_x_e2, gps_y_e2, markersize=5, label=f'GPS error bars')
+        ax_map.legend()
+
+        ax2 = fig.add_subplot(gs[0, 2])
+        ax3 = fig.add_subplot(gs[1, 2])
+        ax2.plot(must_data['sim time'], must_data['speed'], label='MUST cal speed')
+        ax2.plot(gps_data['sim time'], gps_data['speed'], label='GPS speed')
+        ax2.plot(must_uw_data['sim time'], must_uw_data['speed'], label='MUST uw speed')
+        ax2.plot(must_data['sim time'], must_data['speed_interpolated'], label='MUST cal speed smoothed')
+        # ax2.plot(must_data['sim time'], must_data['speed_interpolated'], label='MUST speed (interpolated)')
+        ax2.legend()
+        ax3.plot(must_data['sim time'], must_data['heading'], label='MUST cal heading')
+        ax3.plot(gps_data['sim time'], gps_data['heading'], label='GPS heading')
+        ax3.plot(must_uw_data['sim time'], must_uw_data['heading'], label='MUST uw heading')
+        ax3.plot(must_data['sim time'], must_data['heading_interpolated'], label='MUST cal heading smoothed')
+        # ax3.plot(must_data['sim time'], must_data['heading_interpolated'], label='MUST heading (interpolated)')
+
+        ax2.set_title('speed vs. time')
+        ax2.set_xlabel('time (seconds)')
+        ax2.set_ylabel('speed (m/s)')
+        ax2.set_xlim(must_data['sim time'][0], must_data['sim time'][len(must_data) - 1])
+        ax3.set_title('heading vs. time')
+        ax3.set_xlabel('time (seconds)')
+        ax3.set_ylabel('heading (degrees)')
+        ax3.set_ylim(0, 360)
+        ax3.set_xlim(must_data['sim time'][0], must_data['sim time'][len(must_data) - 1])
+
+        fig.suptitle(f'Test {test_name}')
+        fig = plt.gcf()
+        plt.legend()
+        plt.savefig(os.path.join(output_folder, f'{test_name}_latlon_heading_speed.png'), dpi=100)
+        # plt.show()
+        plt.clf()
+
+    compute_show_metrics(test_name, must_uw_data, gps_data, must_uw_data_matched, gps_data_matched_uw, output_folder, 'uw')
+    compute_show_metrics(test_name, must_data, gps_data, must_data_matched, gps_data_matched, output_folder, 'calibrated')
+
+
+def main(args):
+    base_folder = os.path.join(Path.home(), 'fcp_ws', 'other')
```

Comment:
Can we pass these files and folders are input arguments, and create document how to use this script?